scanners/boostsecurityio/baseline-scanner/module.yaml

```diff
@@ -0,0 +1,24 @@
+api_version: 1.0
+
+
+id: boostsecurityio/baseline-scanner
+name: BoostSecurity Baseline Scanner
+namespace: boostsecurityio/baseline-scanner
```

Comment:
@stlef14 we need to make final choice on the name instead of `BoostSecurity Native Scanner`